num_list.py

```diff
@@ -20,4 +21,5 @@ def print_each_number_and_its_square():
     # ...etc
     # TODO - Write your code below this line.  (Keep the indentation)
     
-
+for i in nums()
```

Comment:
Check your indentation on this block of code as it is "outside" of the `print_each_number_and_its_square` function definition.